__STDC_WANT_LIB_EXT1__ is needed for memset_s and rsize_t on Solaris.

https://bz.apache.org/bugzilla/show_bug.cgi?id=66623

--- crypto/apr_crypto.c
+++ crypto/apr_crypto.c
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#define __STDC_WANT_LIB_EXT1__ 1
 #include <ctype.h>
 #include <stdio.h>
 
